README.md

```diff
@@ -90,6 +90,8 @@ pipenv install --requirements deploy/requirements.txt
 sh scripts/undockered.sh
 ```
 
+Note! To run additional pipenv commands like `pipenv run ./joplin/manage.py migrate` you must have the variable `export DATABASE_URL=postgres://joplin@127.0.0.1:5433/joplin` in your bash scope.
```

Comment:
interesting. what goes wrong if not? Couldn't we make this variable part of the undockered script?

Reply:
It is part of the undockered script. But if you want to just run manage.py commands outside of the script you have to include a DATABASE_URL. Otherwise it'll start building a sqlite database and running migrations on that.